coreos-base/coreos-metadata/files/coreos-metadata.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=CoreOS Metadata Agent
+Requires=network-online.target
+After=network-online.target
```

Comment:
This is the first time we are depending on network-online.target and I am suspicious of its utility and safety... and may not be required for all OEMs anyway. Perhaps fine for now though...